lib/ansible/module_utils/ec2.py

```diff
@@ -119,6 +119,19 @@ def _boto3_conn(conn_type=None, resource=None, region=None, endpoint=None, **par
 
 boto3_inventory_conn = _boto3_conn
 
+
+def boto_exception(err):
```

Comment:
so actually I misread the file name (didn't see it was in modules_utils), this would be the right place for such a function. Unsure if other modules would benefit from it, I don't mind it there or in your ecr code.